addons/website/static/src/js/menu/translate.js

```diff
@@ -146,8 +146,10 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         var current = document.createElement('a');
         current.href = window.location.toString();
         current.search += (current.search ? '&' : '?') + 'enable_editor=1';
-        // we are in translate mode, the pathname starts with '/<url_code/'
-        current.pathname = current.pathname.substr(Math.max(0, current.pathname.indexOf('/', 1)));
+        // We are in translate mode, the pathname starts with '/<url_code'. By
+        // appending a forward slash we can simply search for the first slash
+        // after the language code to cut out the language part.
+        current.pathname = current.pathname.substr(Math.max(0, (current.pathname + '/').indexOf('/', 1)));
```

Comment:
This is probably the correct fix if we want to keep the same line style, but I don't really like that line, I feel like it is hard to read -> a substring of a math.max from a path which receive a triling slash and an index of, no way to figure what it does without reading twice the line.

If you agree, what about: (which seems to do the same, isn't it?)
```js
current.pathname = current.pathname.replace(/^\/[a-zA-Z_-]+\//, '/');
```
(I am the opposite of a regex fan but in this case..)